internal/plugin/command_search.go

```diff
@@ -16,13 +16,13 @@ import (
 )
 
 type ManifestOut struct {
-	Name         string `human:"Name" serialized:"name"`
+	Id           string `human:"Id" serialized:"name"`
```

Comment:
the change to id here was to maintain consistency with the `plugin list` command as we added a new column to the list output named ID, which would show the same content as that of this particular column of the `plugin search` command.

Reply:
Makes sense. Should we show both "ID" and "Name" here in a similar manner, so there's no breaking change. To be clear, the breaking change here is removing the "Name" column?

Reply:
i see. let me discuss this with steven on call and make the change accordingly

Reply:
I think it's a breaking change either way actually, since the current `Name` field for search is the `ID` for the list, install and uninstall.

We can switch this to ID and add in a new Name field to match the list command's Name field but that would still be a breaking change, so I think we should just merge this into V4